test/test_owners.csv

```diff
@@ -601,7 +601,7 @@ k8s.io/kubernetes/pkg/kubelet/dockertools,deads2k,1
 k8s.io/kubernetes/pkg/kubelet/envvars,karlkfi,1
 k8s.io/kubernetes/pkg/kubelet/eviction,childsb,1
 k8s.io/kubernetes/pkg/kubelet/images,caesarxuchao,1
-k8s.io/kubernetes/pkg/kubelet/lifecycle,david-mcmahon,1
+k8s.io/kubernetes/pkg/kubelet/lifecycle,caesarxuchao,1
```

Comment:
Anything with a ,1 at the end was randomly assigned to a maintainer.

Feel free to delete those lines, change some of the other randomly assigned
lines to the correct people (with a 0 to indicate manual assignment), and
run hack/update_owners.py.

On Aug 3, 2016 10:56 AM, "Chao Xu" <notifications@github.com> wrote:

> In test/test_owners.csv
> <https://github.com/kubernetes/kubernetes/pull/29956#discussion_r73380432>
> :
>
> > @@ -601,7 +601,7 @@ k8s.io/kubernetes/pkg/kubelet/dockertools,deads2k,1
> >  k8s.io/kubernetes/pkg/kubelet/envvars,karlkfi,1
> >  k8s.io/kubernetes/pkg/kubelet/eviction,childsb,1
> >  k8s.io/kubernetes/pkg/kubelet/images,caesarxuchao,1
> > -k8s.io/kubernetes/pkg/kubelet/lifecycle,david-mcmahon,1
> > +k8s.io/kubernetes/pkg/kubelet/lifecycle,caesarxuchao,1
>
> Perhaps should be owned by @yujuhong <https://github.com/yujuhong> or
> other members in the node team?
>
> Do you know how this file is generated? I see my name in a few places and
> they don't make sense.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29956/files/029ff3cb7a21eca857bd14079582f9e65a108d0f#r73380432>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAM7nEVv7LxDAPYwd7mHZUOrFimlDdyMks5qcNU_gaJpZM4JbJ4P>
> .
>